TestON/tests/PingallExample/PingallExample.params

```diff
@@ -1,12 +1,16 @@
 <PARAMS>
-    <testcases>1,2,3</testcases>
+    <testcases>1</testcases>
     <ENV>
-        <cellName>kelvin</cellName>
+        <cellName>fiveNodes</cellName>
     </ENV>
```

Comment:
We should try to minimize checking in changes to this example test, especially if it is just test topology changes